lib/core/shell/src/lib/shell.module.ts

```diff
@@ -16,19 +16,11 @@
  */
 
 import { ModuleWithProviders, NgModule } from '@angular/core';
-import { Routes, provideRoutes, Route } from '@angular/router';
-import { SHELL_LAYOUT_ROUTE } from './shell.routes';
-import { ShellLayoutComponent } from './components/shell/shell.component';
+import { Routes, provideRouter } from '@angular/router';
+import { AppShellRoutesConfig, SHELL_LAYOUT_ROUTE } from './shell.routes';
 
-export interface AppShellRoutesConfig {
-    shellParentRoute?: Route;
-    shellChildren: Routes;
-}
-
-@NgModule({
-    imports: [ShellLayoutComponent],
-    exports: [ShellLayoutComponent]
-})
+/** @deprecated use `provideShellRoutes` instead */
```

Comment:
I won't add comment for each place but could you please create jira tasks for each deprecated place from that PR to remove it in the future?

Reply:
Creating a separate jira for deprecated places will produce a lot of spam that nobody is going to pick up in jira. We have over 88 files and 100 cases upon Angular migrations, are they tracked somewhere?

<img width="1033" alt="Screenshot 2025-07-08 at 8 02 12 AM" src="https://github.com/user-attachments/assets/bb8f6517-2886-4edd-9215-7314d36422f3" />